rasa/core/utils.py

```diff
@@ -25,7 +25,7 @@
 from sanic.request import Request
 from sanic.views import CompositionView
 
-from rasa.core.constants import DEFAULT_REQUEST_TIMEOUT
+from rasa.utils import EndpointConfig
```

Comment:
did you speak with Tom about that? We could also change all core imports to `rasa.utils`

Reply:
No, did not talk to Tom. Not quite sure if I understand you... Can you please explain.

Reply:
There a still imports like `from rasa.core.utils import EndpointConfig`. We could also change them to `from rasa.utils import EndpointConfig`.